examples/train_model_cluster.py

```diff
@@ -0,0 +1,34 @@
+"""
+The script recognizes .
+Tom Stuttard
+"""
+
+import os
+from graphnet.utils.cluster_params import *
```

Comment:
I would rename `src/graphnet/utils` -> `src/graphnet/utilities` (since the latter already exists), and in `src/graphnet/utilities/__init__.py` add the line
```
from .cluster_submitter import ClusterSubmitter
```
(provided `cluster_params.py` is renamed to `cluster_submitter.py`) such that the below syntax becomes possible
```suggestion
from graphnet.utilities.cluster import ClusterSubmitter
```

Reply:
I agree. This is a more elegant solution along with the directory cleaning.